_inc/client/plans/single-product-backup.scss

```diff
@@ -1,3 +1,21 @@
+@import '../scss/variables/_colors.scss';
+
+.plans-section__header {
+	color: #23282d;
```

Comment:
We don't need it here - by default all headings in wp-admin are `#23282d` and it's not overridden in Jetpack.